### ISSUE TITLE
Follow system theme

### ISSUE THREAD

**Issue by paramaggarwal** [yep/Nightshift#3]
Is there a way to make this follow the system theme? When macOS is in Dark Mode, then the webpages are dark.

**Comment by iosdevben** [comment]
Seems like the most important feature to support.

**Comment by Volodymyr-13** [comment]
@yep can you please update us with this?

**Comment by tfguinan** [comment]
I had a go [here](https://github.com/tfguinan/Nightshift) and got it kinda working. Aside from requiring building in Xcode, it reloads the page when coming out of dark mode. My first experience with JS so kinda janky. There's probably a better way to do it using the dark reader component.


https://github.com/user-attachments/assets/f214ed0e-e1e2-4a60-aca4-79cd80111b3f

**Comment by Anywhere-Music-Player** [comment]
Well, reloading page is really bad idea, since user could type something or filling form.. or play music and reload will kill it. deal breaker.

**Comment by tfguinan** [comment]
You're right, it should probably call darkReader.disable() instead, though not sure if that will work with the fallback mode. Will keep playing with it when I get some free time.
